### PR TITLE
write unit test statistic

### PR DESCRIPTION
**Pull request by ghost**
## Description
write unit test statistic
## Reference Requirements
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Comment by at-dungle** [comment]
LGTM

File: tests/Browser/Pages/Backend/Home/Statistic.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    public function setUp()
+    {
+        parent::setUp();
+        factory('App\User', 'admin', 1)->create();
+        factory(User::class, self::NUMBER_RECORD_CREATE)->create();
+        factory(Order::class, self::NUMBER_RECORD_CREATE)->create();
+    }
+
+    /**
+     * A Dusk test example.
```
Comment (at-tuanphan) [inline tests/Browser/Pages/Backend/Home/Statistic.php]: update lại comment em ơi

File: tests/Browser/Pages/Backend/Home/Statistic.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    public function testRouteHome()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(User::find(1))
+                    ->visit('/admin')
+                    ->assertSee('TOTAL PRODUCTS', 'TOTAL ORDERED', 'Latest Orders');
+        });
+    }
+
+        /**
```
Comment (at-tuanphan) [inline tests/Browser/Pages/Backend/Home/Statistic.php]: lui vào 1 tab em

File: tests/Browser/Pages/Backend/Home/Statistic.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    /**
+     * A Dusk test example.
+     *
+     * @return void
+     */
+    public function testRouteHome()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(User::find(1))
```
Comment (at-tuanphan) [inline tests/Browser/Pages/Backend/Home/Statistic.php]: User::find(1) => $this->user

File: tests/Browser/Pages/Backend/Home/Statistic.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+        /**
+     * Test value show on page.
+     *
+     * @return void
+     */
+    public function testValueOnHomePage()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(User::find(1))
```
Comment (at-tuanphan) [inline tests/Browser/Pages/Backend/Home/Statistic.php]: User::find(1) => $this->user

**Review by ghost — COMMENTED** [review]
please check my comments

**Comment by at-sonle** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**